src/wix/test/WixToolsetTest.BuildTasks/WixToolsetTest.BuildTasks.csproj

```diff
@@ -22,6 +22,12 @@
     <ProjectReference Include="..\..\WixToolset.Core.TestPackage\WixToolset.Core.TestPackage.csproj" />
   </ItemGroup>
 
+  <!-- Copy the x64 binaries for unittests since we've standardized on VS2022 (which is 64-bit only) -->
```

Comment:
I think it's more correct to say VS2022 is 64-bit by default. There's still settings to go to 32-bit, like there were settings to go to 64-bit in earlier versions.